src/hooks/plans/useCreateEditPlan.ts

```diff
@@ -48,7 +48,8 @@ gql`
       packageSize
       rate
       fixedAmount
-      fixedAmountTarget
+      freeUnitsPerEvents
+      freeUnitsPerTotalAggregation
```

Comment:
This one is not used here

Reply:
As discussed together, I created a refactor issue to fix this (and other in this file) later

https://github.com/getlago/lago-front/issues/354